From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matt Clarkson <mattyclarkson@gmail.com>
Date: Fri, 25 Oct 2024 19:01:29 +0100
Subject: [PATCH] build(bazel): modify `#include` from `<>` to `""`

Some internal headers are included as system includes `<>` rather than
local `""`.
---
 krl.c                           | 4 ++--
 openbsd-compat/readpassphrase.c | 2 +-
 openbsd-compat/setproctitle.c   | 2 +-
 servconf.h                      | 2 +-
 sftp-usergroup.c                | 2 +-
 srclimit.c                      | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/krl.c b/krl.c
index e2efdf066..f648e1c29 100644
--- a/krl.c
+++ b/krl.c
@@ -18,8 +18,8 @@
 #include "includes.h"
 
 #include <sys/types.h>
-#include <openbsd-compat/sys-tree.h>
-#include <openbsd-compat/sys-queue.h>
+#include "openbsd-compat/sys-tree.h"
+#include "openbsd-compat/sys-queue.h"
 
 #include <errno.h>
 #include <fcntl.h>
diff --git a/openbsd-compat/readpassphrase.c b/openbsd-compat/readpassphrase.c
index ff8ff3dec..2f97d8587 100644
--- a/openbsd-compat/readpassphrase.c
+++ b/openbsd-compat/readpassphrase.c
@@ -31,7 +31,7 @@
 #include <signal.h>
 #include <ctype.h>
 #include <fcntl.h>
-#include <readpassphrase.h>
+#include "readpassphrase.h"
 #include <errno.h>
 #include <string.h>
 #include <unistd.h>
diff --git a/openbsd-compat/setproctitle.c b/openbsd-compat/setproctitle.c
index e4064323a..91727404a 100644
--- a/openbsd-compat/setproctitle.c
+++ b/openbsd-compat/setproctitle.c
@@ -44,7 +44,7 @@
 #endif
 #include <string.h>
 
-#include <vis.h>
+#include "vis.h"
 
 #define SPT_NONE	0	/* don't use it at all */
 #define SPT_PSTAT	1	/* use pstat(PSTAT_SETCMD, ...) */
diff --git a/servconf.h b/servconf.h
index 5089bc9ea..b0290697c 100644
--- a/servconf.h
+++ b/servconf.h
@@ -16,7 +16,7 @@
 #ifndef SERVCONF_H
 #define SERVCONF_H
 
-#include <openbsd-compat/sys-queue.h>
+#include "openbsd-compat/sys-queue.h"
 
 #define MAX_PORTS		256	/* Max # ports. */
 
diff --git a/sftp-usergroup.c b/sftp-usergroup.c
index 93396ffc6..b90fec91e 100644
--- a/sftp-usergroup.c
+++ b/sftp-usergroup.c
@@ -19,7 +19,7 @@
 #include "includes.h"
 
 #include <sys/types.h>
-#include <openbsd-compat/sys-tree.h>
+#include "openbsd-compat/sys-tree.h"
 
 #include <stdlib.h>
 #include <stdarg.h>
diff --git a/srclimit.c b/srclimit.c
index 33116fa52..935d254a0 100644
--- a/srclimit.c
+++ b/srclimit.c
@@ -19,7 +19,7 @@
 
 #include <sys/socket.h>
 #include <sys/types.h>
-#include <openbsd-compat/sys-tree.h>
+#include "openbsd-compat/sys-tree.h"
 
 #include <limits.h>
 #include <netdb.h>
